stable/swagger/templates/deployment.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
```

Comment:
The `extensions/v1beta1` apiversion will be going away for deployments. It's late in the deprecation cycle. Could you use `apps/v1beta2` instead?